script.js

```diff
@@ -1,4 +1,82 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var opponentSelection = scissorsPaperStone();
+  var lowerCaseOpponentSelection = opponentSelection.toLowerCase();
+  var lowerCaseInput = input.toLowerCase();
+  var myOutputValue =
+    "Sorry, you lost! You selected " +
+    input +
+    " but the opponent selected " +
+    opponentSelection +
+    ". Please try again!";
```

Comment:
Excellent choice of setting the default `myOutputValue` to one of the following conditions: win, or loss.

Not entirely sure if that's a deliberate decision on your end, but most students end up writing separate yet bloated conditions for checking both the win and loss condition.

There are 4 conditions to account for in this Project: invalid input, win, loss, and draw. Invalid input and draw are very specific conditions, that look very dissimilar from the other conditions.

Unlike invalid input and draw, winning and losing are ultimately very similar conditions. Winning is just the opposite of losing, no? Thus, when writing conditionals for scissors, paper, stone, there's actually no need to write the conditionals for both winning and losing. You need only 1.

Glad that you seem to be able to recognise it, and have the loss condition as the default! Thus, you need only write out the logic (`if` condition) for handling wins.